code/modules/client/preference_setup/global/preferences.dm

```diff
@@ -248,9 +248,9 @@ var/global/list/_client_preferences_by_type
 	key = "EXAMINE_MESSAGES"
 	options = list(PREF_SHOW, PREF_HIDE)
 
-/datum/client_preference/floating_messages
-	description = "Floating chat messages"
-	key = "FLOATING_CHAT"
+/datum/client_preference/runechat
+	description = "Show Runechat (Above-Head-Speech)"
+	key = "RUNECHAT_SHOW"
```

Comment:
Don't really like changing this preference, it means people's old prefs will be invalidated and the description is silly, there's no reason to call it Runechat in the pref if you then have to explain it anyway. Keep it as 'show floating speech' and keep the key imo.